$NetBSD$

- fix lib creation rule (apparently does not work with some gmake
versions as it has been failing in Linux bulk builds)

--- Rules.in~	2001-09-06 12:41:36.000000000 +0000
+++ Rules.in
@@ -52,7 +52,10 @@ depend:       
 	@$(CC) $(CFLAGS) $(XCFLAGS) -M $(OBJS:.o=.c) $(XDEPS:.o=.c) >$@
 
 $(OBJS): $(MAKEFILE)
+
+ALWAYS: ;
 
-$(LIB): $(LIB)($(OBJS) $(XOBJS))
+$(LIB): $(OBJS) $(XOBJS) ALWAYS
+	$(AR) -cruv $(LIB) $(OBJS) $(XOBJS)
 	$(RANLIB) $(LIB)
 
 subdist:
